### ISSUE TITLE
After running for a period of time, the loss is nan

### ISSUE THREAD

**Issue by datakong** [AkaliKong/iLoRA#6]
Dear author:
I also discovered the issue of losing Nan after running your code for a period of time during the process of reproducing it. Another work, LLaRa, did not have this problem. Looking forward to your reply! Wishing you a smooth scientific research!

**Comment by AkaliKong** [comment]
Thank you for bringing this matter to my attention. I sincerely apologize if this situation has affected your research progress.

Based on my initial assessment, the issue might be related to the increased complexity of the ilora structure, which could require adjustments to the learning rate. I believe using a smaller learning rate might resolve the problem you've encountered. Another potential solution would be to implement gradient clipping.

I will carefully consider the issue you've raised and will update my GitHub repository as soon as possible with the necessary adjustments.

Thank you for your understanding and patience.

**Comment by datakong** [comment]
Hello, respected author. Thank you very much for your reply. I noticed that the learning rate of train_lastfm.sh is different from that described in the paper. Is this the reason?

**Comment by 2000lxylxy** [comment]
> Hello, respected author. Thank you very much for your reply. I noticed that the learning rate of train_lastfm.sh is different from that described in the paper. Is this the reason?

Have you solved this problem regarding NAN values? I'm stuck with the same problem.

**Comment by Cheng-lg** [comment]
Have you solved this problem regarding NAN values? I'm stuck with the same problem.
